packages/cli/src/TagHelpers.ts

```diff
@@ -86,3 +88,67 @@ export async function removeRelations(workflowId: string, tablePrefix: string) {
 		.where('workflowId = :id', { id: workflowId })
 		.execute();
 }
+
+const createTag = async (transactionManager: EntityManager, name: string): Promise<TagEntity> => {
+	const tag = new TagEntity();
+	tag.name = name;
+	return transactionManager.save<TagEntity>(tag);
+};
```

Comment:
Subjective, but since this is very readable and only three lines and not reused, I wouldn't abstract it away into a separate helper.

Reply:
just helps readability